optionals/sys_spe/SPE_Tanks.hpp

```diff
@@ -0,0 +1,565 @@
+    acre_infantryPhoneIntercom[] = {"all"}; // all or list of intercoms that are connected to the phone
+
+    class AcreIntercoms {
+        class Intercom_1 {
+            displayName = CSTRING(Intercom_Crew);
+            shortName = CSTRING(Intercom_Short);
+            allowedPositions[] = {"crew"};
+            disabledPositions[] = {};
+            limitedPositions[] = {};
+            masterPositions[] = {"commander"};
+            numLimitedPositions = 0;
+            connectedByDefault = 1;
+        };
+    };
+
+    class AcreRacks {
+        class Rack_1 {
+            displayName = CSTRING(RadioSet);
+            shortName = CSTRING(RadioSetShort);
+            mountedRadio = "ACRE_PRC77"; // This is the only analog radio currently in ACRE and is the best fit for the WW2 time perios.
+            componentname = "ACRE_VRC64"; // This is the rack that the radio attaches to, PRC 77 requires the VRC 64.
+            defaultComponents[] = {}; // Not yet fully implemented in ACRE
+            allowedPositions[] = {"crew"}; // which positions have full access to the radio, including listening (Rx), speaking (Tx), and configuration.
+                                           // In the base config here we're allowing the entire crew, which will ensure that all tanks have a functional radio even if a custom implementation hasn't been created below.
+            disabledPositions[] = {{"turnedout", "all"}}; // by default if you turn out you can't use the radio anymore
+            isRadioRemovable = 0; // 0 prevents the radio from being removed and used as an infantry radio.
+            intercom[] = {"intercom_1"}; // list of intercoms that can talk over the radio even if not a slot in allowedPositions
+        };
+    };
+
+    class Turrets;
+};
+
+    // American
+class SPE_M10_base : SPE_Tank_base {
+    class AcreRacks {
+        class Rack_1 {
+            allowedPositions[] = {"commander", {"turret", {1}}}; // restrict configuration of the radio to only and radio operator (historically correct) and the the commander (for gameplay, even though the radio was in the hull)
+        };
+    };
+};
+
+class SPE_M18_Hellcat_Base : SPE_Tank_base {
+    class AcreRacks {
+        class Rack_1 {
+            allowedPositions[] = {"commander", "gunner", {"turret", {0,1}}}; // restrict radio configuration to the commander, gunner, and loader, as the radio was in the turret
+        };
+    };
+};
+
+class SPE_Sherman_base : SPE_Tank_base {
+    acre_hasInfantryPhone = 1; // The Sherman was the first tank to get an infantry phone
+    acre_infantryPhoneDisableRinging = 1;
+    acre_infantryPhonePosition[] = {0, -2.52, 0};
+
+    class AcreRacks {
+        class Rack_1 {
+            allowedPositions[] = {"commander", "gunner", {"turret", {0,1}}}; // commander, gunner, and loader, radio in turret
+        };
+    };
+};
+
+class SPE_M4A1_75;
+class SPE_M4A1_75_Command : SPE_M4A1_75 {
+    displayName = CSTRING(M4A1_75_Command);
+    scope = 2;
```

Comment:
are these vehicles we're creating?
do they need to be in `units[]`?

Reply:
Ahh yeah they should be in units for the American ones.